packages/mattermost-redux/src/reducers/entities/emojis.ts

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {combineReducers} from 'redux';
+
+import {EmojiTypes, PostTypes, UserTypes} from 'mattermost-redux/action_types';
+import {EmojisState, CustomEmoji} from 'mattermost-redux/types/emojis';
+import {GenericAction} from 'mattermost-redux/types/actions';
+import {Post} from 'mattermost-redux/types/posts';
+import {IDMappedObjects} from 'mattermost-redux/types/utilities';
+
+export function customEmoji(state: IDMappedObjects<CustomEmoji> = {}, action: GenericAction): IDMappedObjects<CustomEmoji> {
```

Comment:
There were some weird typings here before. Most of these were imported as `import * as types from 'types'` which meant all of them were referred to as things like `types.entities.CustomEmoji`. It wasn't strictly necessary to fix if I updated the import path, but I couldn't help myself.